showcase/showcase-app.js

```diff
@@ -1,47 +1,86 @@
 import React, {Component} from 'react';
-import AxesShowcase from './showcase-sections/axes-showcase';
-import PlotsShowcase from './showcase-sections/plots-showcase';
-import SunburstSection from './showcase-sections/sunburst-showcase';
-import RadialShowcase from './showcase-sections/radial-showcase';
-import RadarShowcase from './showcase-sections/radar-showcase';
-import LegendsShowcase from './showcase-sections/legends-showcase';
-import SankeysShowcase from './showcase-sections/sankeys-showcase';
-import TreemapShowcase from './showcase-sections/treemap-showcase';
-import MiscShowcase from './showcase-sections/misc-showcase';
-
 import PropTypes from 'prop-types';
+import ShowcaseDropdown from './showcase-components/showcase-dropdown';
+
+import ShowcaseIndex from './showcase-index';
```

Comment:
Can't we import directly the separate components rather than destructuring this?